google/cloud/aiplatform/training_jobs.py

```diff
@@ -26,6 +26,7 @@
 import time
 from typing import Callable, List, Optional, Sequence, Union
 
+from abc import ABC, abstractmethod
```

Comment:
Module level import here.

Reply:
I was wondering about this. So according to the style guide I should be import abc, and then refer use `abc.abstractmethod` every time in the code?

What about the line above it? 
`from typing import Callable, List, Optional, Sequence, Union`

Reply:
Yes on `abc.abstractmethod`.

`typing` is an exception to the rule [typing-imports](https://engdoc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#typing-imports).